portfolio/src/main/webapp/script.js

```diff
@@ -112,18 +112,28 @@ window.addEventListener('DOMContentLoaded', (event) => {
     showSlides(slideIndex);
 });
 
-function getMessage() {
-  fetch('/data').then(response => response.json()).then((comment) => {
-
-    const commentElement = document.getElementById('greeting');
-    commentElement.innerHTML = '';
-    commentElement.appendChild(
-        createListElement(comments[0]));
+function getComments() {
+  const limit = document.getElementById('input').value;
+  fetch('/data?limit-input=' + limit).then(response => response.json()).then((comments) => {
+    const commentElement = document.getElementById('comments-container');
+    commentElement.innerHTML = "";
+    comments.forEach((comment) => {
+      commentElement.appendChild(createCommentElement(comment));
```

Comment:
Pulling `createCommentElement()` out into its own function makes this much easier to follow, awesome!